guides/common/modules/con_using-keyboard-shortcuts.adoc

```diff
@@ -0,0 +1,8 @@
+[id="using-keyboard-shortcuts_{context}"]
+= Using Keyboard Shortcuts
+
+You can use keyboard shortcuts to quickly focus search bars.
+
+To focus the vertical navigation search bar, press `CTRL + SHIFT + F`.
+
+To focus the page search bar, press `/`.
```

Comment:
Can you please try https://docs.asciidoctor.org/asciidoc/latest/macros/keyboard-macro/?

Also, maybe make line 6+8 an unordered list?

Reply:
Seems like the macro does not work:
![image](https://github.com/theforeman/foreman-documentation/assets/108661422/e62fb774-441b-498b-8292-37cfa560bba5)
so I am reverting that.

Reply:
Ah, I remember that they need ":experimental:" set. Probably not something we want to run in prod right now. See https://docs.asciidoctor.org/asciidoc/latest/macros/keyboard-macro/

Thanks for testing anyway.

Reply:
Hmm, I actually did not notice that. To be honest, I wouldn't mind having the macro in the module, it does say that the macro is stable. Shall I add it back?

Reply:
There is a catch. They have also mentioned that they might change the name of attribute (:experimental:) after their internal decision. We can delay adding macro?

Reply:
If it's not cherry-picked, then trying it out is really OK for me. Let's see what happens.

Reply:
That as fair enough. I will merge as-is after QE review.